packages/mrm-task-lint-staged/__snapshots__/index.spec.js.snap

```diff
@@ -221,13 +191,32 @@ exports[`should merge ESLint and Prettier into a single lint-staged rule 1`] = `
   \\"lint-staged\\": {
     \\"*.js\\": [
       \\"eslint --fix\\",
-      \\"prettier --write\\",
-      \\"git add\\"
+      \\"prettier --write\\"
     ]
   }
 }"
 `;
 
+exports[`should remove \`git add\` from lint-staged config if lint-staged v10 is used 1`] = `
+Object {
+  "/package.json": "{
+  \\"name\\": \\"unicorn\\",
+  \\"scripts\\": {},
+  \\"devDependencies\\": {
+    \\"eslint\\": \\"*\\"
+  },
+  \\"lint-staged\\": {
+    \\"*.js\\": \\"eslint --fix\\"
```

Comment:
There was `some other command` right?

Reply:
That's a great catch. I was sure it looked good in tests. That's why I dislike snapshots 🤷‍♂